src/officehours_api/tests.py

```diff
@@ -0,0 +1,146 @@
+from unittest import mock
+
+from django.test import TestCase, override_settings
+
+from officehours_api.models import User, Queue, Meeting
+
+
+@override_settings(TWILIO_ACCOUNT_SID='aaa', TWILIO_AUTH_TOKEN='bbb', TWILIO_PHONE_FROM='+15555552323')
+class NotificationTestCase(TestCase):
+    def setUp(self):
+        self.foo = User.objects.create(username='foo', email='foo@example.com')
+        self.foo.profile.phone_number = '+15555550000'
+        self.foo.profile.notify_me_attendee = True
+        self.foo.profile.notify_me_host = True
```

Comment:
It feels like there should be a way to make this more concise, like maybe this?

```
self.foo.profile = Profile(phone_number='...', notify_me_attendee=True, notify_me_host=True)
self.foo.save()
```
or
```
some_profile = Profile(user=self.foo, phone_number='...', notify_me_attendee=True, notify_me_host=True)
```
Just an idea, I haven't tried it this with a 1:1 field before.

Reply:
Good idea, but I couldn't get that or a close variation of it (`= Profile(user_id=self.foo.id`, `= Profile.objects.create(`, ...) to work, I think owing to the fact the profile is autogenerated for the user post-save. Instead, I condensed it with a very 1:1 refactoring.

Reply:
Yeah, that works, thanks for looking into it!